mu4/notation/internal/notationinteraction.cpp

```diff
@@ -1682,6 +1684,44 @@ void NotationInteraction::drawTextEditMode(QPainter* painter)
     m_textEditData.element->drawEditMode(painter, m_textEditData);
 }
 
+void NotationInteraction::drawSelectionRange(QPainter* painter)
+{
+    if (!m_selection->isRange()) {
+        return;
+    }
+
+    painter->setBrush(Qt::NoBrush);
+
+    QPen pen;
+    pen.setColor(MScore::selectColor[0]);
```

Comment:
I think better to add a method to configuration:

QColor NotationConfiguration::selectionColor() { return MScore::selectColor[0] }